source/release-notes/v1.6-release-notes.rst

```diff
@@ -0,0 +1,178 @@
+.. _v1.6-release-notes:
+
+v1.6 Release Notes
+==================
+
+Highlights in 1.6:
+
+- `OnDemand passed a security audit!`_
+- `Added job array support for LSF and PBSPro adapters`_
+- `Added L18n hooks to the Dashboard and Job Composer`_
+- `Added rendering of HTML or Markdown in Job Composer template manifests`_
+- `Changed ngnix_stage to always remove stale PID files`_
+- `Fixed Active Jobs to show empty cell if account is nil`_
```

Comment:
Instead of having separate sections for each bug fix and each adapter improvement lets combine these into two sections:

1. Fixed several bugs
2. Improved SGE and Slurm adapters

The content of the original sections can be added to each, as separate paragraphs, or in the case of the bug fixes, as bulleted list